sdk/lib/rtl/condvar.c

```diff
@@ -456,13 +456,15 @@ InternalSleep(IN OUT PRTL_CONDITION_VARIABLE ConditionVariable,
 }
 
 VOID
+NTAPI
```

Comment:
Why does these two functions require NTAPI? Is it because they are exported?

Reply:
I isnt technically required, it's just what we usually do and when I first added a prototype to ntdllp.h, I intuitively declared it as NTAPI, which didn't match, so I went with changing it to be NTAPI.